metrics.md

```diff
@@ -0,0 +1,226 @@
+# Side-Car Metrics
```

Comment:
Instead of everything being prefixed would "side-car"

Why don't we just use "Slinky"?